articles/building-a-simple-cryptocurrency-blockchain/index.md

```diff
@@ -0,0 +1,175 @@
+## Building Simple cryptocurrency system using Node.js
+
+In this tutorial, we are going to demonstrate how to build a simple cryptocurrency system, called thecoin.
+![Image title](engineering-education/building-a-simple-cryptocurrency-blockchain/trade.jpg)
+
+We are going to implement the concepts of JavaScript classes and Node.js.
+
+## Prerequisite
+To follow this tutorial successfully, you’ll need to have the following:
+
+- Node.js Installed in your machine
+
+- Code editor for my case I use VSCode
```

Comment:
Also, mention about the technical prerequisites like what concepts would enable the reader to understand this topic better